packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -3,20 +3,24 @@ description: A common platform interface for the url_launcher plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.8
+version: 2.0.0-nullsafety
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.0.5
-  plugin_platform_interface: ^1.0.1
+  # TODO (mvanbeusekom): use pub.dev once 1.10.0-nullsafety released.
+  plugin_platform_interface:
+    git: 
```

Comment:
It looks like this would need to be a local path dependency: https://api.cirrus-ci.com/v1/task/4983717032624128/logs/main.log

Reply:
I have updated the code to use a local path dependency which has the same result unfortunately (https://api.cirrus-ci.com/v1/task/6308195557703680/logs/main.log). 

We can only solve this be publishing the null safety version of the "plugin_platform_interface" package (or of course updating the publishable CI step, which is highly undesirable in my opinion).

Reply:
I see. I think we can skip this test for now. Feel free to revert it back to a git dependency. I think we will need that for some testing in the Flutter framework repo.

Reply:
Thanks, I have reverted back to the git dependency just to be sure.